experimental/traffic-portal/src/app/core/servers/server-details/server-details.component.html

```diff
@@ -189,7 +189,7 @@
 				</div>
 			</fieldset>
 			<div class="buttons">
```

Comment:
submit button isn't working.

Reply:
Submit button _"works"_ just fine, problem is the controller has no handler for editing servers. It only creates them; which is a bug not caused by this PR.

Reply:
Ok, creating an [issue](https://github.com/apache/trafficcontrol/issues/7485) for the same.